### PR TITLE
fix(chat): Change 'Continue' button white font on light theme in import duplicates dialog (Issue #1235)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Button Continue on import duplicates dialog should have white font on light theme

Issues:

- #1235 

**UI changes**

https://www.figma.com/design/lwL5jneveZczAj5RLcLFH2/DIAL-Chat?node-id=0-1&t=pkZn9CzSaqBvxvAw-0

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1384.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1384.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx [hunk 1]
```diff
@@ -337,7 +336,7 @@ export const ReplaceConfirmationModal = ({ isOpen }: Props) => {
           </button>
           <button
             onClick={handleContinueImport}
-            className="h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover"
+            className="button button-primary h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover"
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: Are you sure that other old classes are required here?
`h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover`

Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: I didn’t try to touch the old codes, I just added what was in the task, if necessary I can look

Comment (Alexander-Kezik) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: `px-3` and `rounded` are not required here. They are already present in the .button class. Could you please remove them?

Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: Yes sure, Thank you for the clarification

Comment (IlyaBondar) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: @Derikyan ,
`h-[38px] rounded bg-controls-accent px-3 hover:bg-controls-accent-hover`
image

Comment (Derikyan) [reply apps/chat/src/components/Common/ReplaceConfirmationModal/ReplaceConfirmationModal.tsx]: "button button-primary h-[38px]" should stay there, thanks

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1384.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Alexander-Kezik — APPROVED** [review]

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1384.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1384.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**